sdk/src/store.rs

```diff
@@ -2613,15 +2586,15 @@ impl Store {
         signer: &dyn AsyncSigner,
         format: &str,
         asset_reader: Option<&mut dyn CAIRead>,
-        settings: &Settings,
+        context: &Context,
     ) -> Result<Vec<u8>> {
         let mut jumbf_bytes =
             self.prep_embeddable_store(signer.reserve_size(), dh, asset_reader)?;
 
         // sign contents
         let pc = self.provenance_claim().ok_or(Error::ClaimEncoding)?;
         let sig = self
-            .sign_claim_async(pc, signer, signer.reserve_size(), settings)
+            .sign_claim_async(pc, signer, signer.reserve_size(), context.settings())
```

Comment:
What if we would pass in the whole context here instead?

Reply:
There are a lot of tradeoffs in where to pass context vs settings vs individual values. I'm not trying to fully answer that in this PR, just trying to inject the Context at the higher level so we can start using it.